ci/lint.yml

```
@@ -5,8 +5,8 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: golang
-    tag: 1.24.1-bullseye
+    repository: onsdigital/dp-concourse-tools-lint-go
+    tag: 1.24.4-bullseye-golangci-lint-2
```

It looks like this can increase by one more
```suggestion
    tag: 1.24.5-bullseye-golangci-lint-2
```